src/main/java/com/github/dockerjava/api/model/PropagationMode.java

```diff
@@ -0,0 +1,49 @@
+package com.github.dockerjava.api.model;
+
+/**
+ * The propagation mode of a file system or file: <code>shared</code>, <code>slave</code> or <code>private</code>.
+ *
+ * @see https://github.com/docker/docker/pull/17034
+ * @since 1.22
+ */
+public enum PropagationMode {
+    /** default */
+    mount_default(""),
```

Comment:
I renamed the values but could not figure out where the JsonProperty annotation should go here. My code should be in line with `AccessMode` and `SELContext`.